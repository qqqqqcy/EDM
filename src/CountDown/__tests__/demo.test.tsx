import webDemoTest from '@tests/shared/demoTest';
webDemoTest('CountDown');
