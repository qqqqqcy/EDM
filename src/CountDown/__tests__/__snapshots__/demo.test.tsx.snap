// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CountDown correctly 1`] = `
<div>
  <div>
    <h5>
      倒计时1天
    </h5>
    <div
      class="edm-countdown"
    >
      <span>
        01 天 
      </span>
      <span>
        00 时 
      </span>
      <span>
        00 分 
      </span>
      <span>
        00 秒 
      </span>
    </div>
    <h5>
      倒计时180秒
    </h5>
    <div
      class="edm-countdown"
    >
      <span>
        00 : 
      </span>
      <span>
        03 : 
      </span>
      <span>
        00 
      </span>
    </div>
    <h5>
      倒计时307秒
    </h5>
    <div
      class="edm-countdown"
    >
      <span>
        00 : 
      </span>
      <span>
        05 : 
      </span>
      <span>
        07 
      </span>
    </div>
    <h5>
      倒计时120秒
    </h5>
    <div
      class="edm-countdown"
    >
      <span>
        120  
      </span>
    </div>
    <h5>
      倒计时5秒
    </h5>
    <div
      class="edm-countdown"
    >
      <span>
        05  
      </span>
    </div>
  </div>
</div>
`;
